src/jjsp/jde/JDE.java

```diff
@@ -590,7 +590,7 @@ protected void refreshTab(boolean gainedFocus, boolean isDisplayed, boolean wasU
                     try
                     {
                         if (ff.lastModified() > jde.getLastModified())
-                            jde.loadAutoSavedData(Utils.getAsciiBytes(JDETextEditor.loadTextFromURI(uri)));
+                            jde.loadAutoSavedData(Utils.getBytes(JDETextEditor.loadTextFromURI(uri)));
```

Comment:
Seems that there is call to `Utils.toString()` in the chain for this line. It uses ASCII charset.
Can you fix that or simplify the chain to just getting the bytes?